ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -156,7 +156,7 @@ class LocalAssemblerData
 
     std::vector<std::vector<double>> _darcy_velocities
         = std::vector<std::vector<double>>(
-            GlobalDim, std::vector<double>(ShapeFunction::NPOINTS));
+            GlobalDim, std::vector<double>(_integration_method.getNumberOfPoints()));
```

Comment:
@chleh It is finely working with clang-3.5; see #1448 (I tested with this PR merged together and the issues is indeed resolved).